synctl

```diff
@@ -3715,13 +3958,56 @@ def main():
             split_string = get_args.custom_property.split(',')
             patch_instance.patch_custom_properties(get_args.id, split_string)
     elif COMMAND_UPDATE == get_args.sub_command:
-        if get_args.file is not None:
-            with open(get_args.file, 'rb') as json_file:
-                payload = json_file.read()
-            if get_args.syn_type == SYN_TEST:
-                syn_instance.update_a_synthetic_test(get_args.id, payload)
-            if get_args.syn_type == SYN_ALERT:
-                syn_instance.update_a_smart_alert(get_args.id, payload)
+        if get_args.syn_type == SYN_TEST:
+            payload = syn_instance.retrieve_a_synthetic_test(get_args.id)
+            if get_args.file is not None:
+                new_payload = update_instance.update_using_file(get_args.file)
+            if get_args.label is not None:
+                new_payload = update_instance.update_label(payload[0], get_args.label)
+            if get_args.active is not None:
```

Comment:
can we set --file option and other options exclusive? they are duplicated as a json file already contain all fields of a test.